src/main/java/com/example/prolect4_test1/game_genre/Game_Genre.java

```diff
@@ -0,0 +1,58 @@
+package com.example.prolect4_test1.game_genre;
+
+import com.example.prolect4_test1.game.Game;
+import com.example.prolect4_test1.genre.Genre;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+
+@Entity
```

Comment:
Why creating the join table manually? since there are no other attributes to the class a many to many relationship between genre and games would be enough.

Also the table name should be plural and the class name should be in Pascal case and not snake case, and lastly there are some extra empty lines in this file